core/src/main/java/io/questdb/griffin/engine/functions/date/ExtractFromTimestampFunctionFactory.java

```diff
@@ -260,6 +219,17 @@ public Function getArg() {
             return arg;
         }
 
+        @Override
+        public String getName() {
```

Comment:
is there a value of removing arg types? We could have single static constant in each factory, getSignature() and getName() would return the same value? What do you think?

Reply:
For functions toPlan() produces actual expression with arguments. Most implementations are fine and match number of actual arg with function type (unary, binary, ... ) but there's a large number of case-specific functions that cache arg and take  fewer arguments (compared to their signature).

Reply:
got it, makes sense!